.eslintrc.js

```diff
@@ -28,6 +28,9 @@ module.exports = {
     }],
     'vue/html-self-closing': 'off',
     "vue/no-v-html": 'off',
+    'vue/singleline-html-element-content-newline': 'off',
+    'vue/require-default-prop': 'off',
+    'vue/require-prop-types': 'off',
```

Comment:
This change was intentional in order to make small PR. I will create separate PR for fixing all the Eslint issues

Reply:
Made it a warning instead of error.